x-pack/plugins/ml/public/application/components/callout/callout.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { EuiCallOut, EuiLink, EuiSpacer } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { VALIDATION_STATUS } from '../../../../common/constants/validation';
+
+export const defaultIconType = 'questionInCircle';
+
+export interface CalloutMessage {
+  id?: string;
+  heading?: string;
+  status?: VALIDATION_STATUS;
+  text: string;
+  url?: string;
+}
+
+const statusToEuiColor = (status: VALIDATION_STATUS) => {
+  switch (status) {
+    case VALIDATION_STATUS.INFO:
+      return 'primary';
+      break;
```

Comment:
nit, the breaks are redundant in these `switch` statements as you're returning from inside the case

Reply:
just noticed this is moved code. might be worth updating it still.

Reply:
Updated in a7e5fc69c41f0b104667516fc53457eebf0a649d